data/css/modules/navigation/_searchBox.scss

```diff
@@ -1,4 +1,11 @@
 .NavigationSearchBox {
+    .WindowSimple & {
```

Comment:
The theme for top bar search search bar actually lives in eos-sdk IIRC. See the `.endless-search-box` theming there.

We want any app using a top bar search box to get this fix, so I think this should land in eos-sdk instead.